src/components/ItemModalBackground.tsx

```diff
@@ -0,0 +1,201 @@
+import { useState, useEffect } from "react";
+import Box from "@mui/material/Box";
+import TextField from "@mui/material/TextField";
+import { DatePicker, LocalizationProvider } from "@mui/x-date-pickers";
+import { AdapterDayjs } from "@mui/x-date-pickers/AdapterDayjs";
+
+export default function ItemModalBackground({
+  setOpenModal,
+}: {
+  setOpenModal: () => void;
+}) {
+  const [formData, setFormData] = useState({});
+  const handleInputChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const { name, value } = event.target;
+    setFormData({ ...formData, [name]: value });
+  };
+
+  const [selectedTags, setSelectedTags] = useState<string[]>([]);
+
+  const handleTagClick = (tag: string) => {
+    const index = selectedTags.indexOf(tag);
+    if (index !== -1) {
+      setSelectedTags(selectedTags.filter((_, i) => i !== index));
+    } else {
+      setSelectedTags([...selectedTags, tag]);
+    }
+  };
+  useEffect(() => {
+    setFormData({
+      ...formData,
+      Categories: selectedTags,
+    });
+  }, [selectedTags]);
+
+  return (
+    <div className="modalBackground">
+      <div className="modalContainer">
```

Comment:
The padding around the fields and tags are not the same throughout. Make sure everything is aligned around all edges